archesproject/arches
# Keep file widget accessibility features when filtering

### Types of changes


### Description of Change
The prior implementation of file filtering in the file widget duplicated a large block of template code. The accessibility features in #9419 only targeted the non-filtered version. Resolve this by DRYing out the code.

### Issues Solved
Closes #9947

### Checklist


#### Ticket Background
*   Sponsored by: 
*   Found by: @jacobtylerwalls
*   Tested by: @jacobtylerwalls
*   Designed by: @ 

### Testing instructions
Upload multiple files to the same file node.
Save edits.
Filter on one of the file names.
Inspect the "download" button, ensure the aria label includes the file name.
